### ISSUE TITLE
Mac GCC compiler error

### ISSUE THREAD

**Issue by K0lb3** [wolfpld/etcpak#5]
Hello,
I made a python wrapper for your project and the following error occurred on all macOS dockers.

```
gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -I/usr/local/opt/sqlite/include -I/usr/local/opt/sqlite/include -Isrc/etcpak -Isrc/etcpak/libpng -Isrc/etcpak/zlib -Isrc/etcpak/lz4 -I/Users/runner/hostedtoolcache/Python/3.9.2/x64/include/python3.9 -c src/etcpak/ProcessDxtc.cpp -o build/temp.macosx-10.14-x86_64-3.9/src/etcpak/ProcessDxtc.o -std=c++11
src/etcpak/ProcessDxtc.cpp:29:37: warning: unused function 'to565' [-Wunused-function]
static etcpak_force_inline uint16_t to565( uint8_t r, uint8_t g, uint8_t b )
                                    ^
src/etcpak/ProcessDxtc.cpp:118:23: warning: unused variable 'DivTableNEON' [-Wunused-const-variable]
static const uint16_t DivTableNEON[255*3+1] = {
                      ^
src/etcpak/ProcessDxtc.cpp:621:37: warning: unused function 'ProcessAlpha' [-Wunused-function]
static etcpak_force_inline uint64_t ProcessAlpha( const uint8_t* src )
                                    ^
src/etcpak/ProcessDxtc.cpp:754:24: error: always_inline function '_mm_broadcastb_epi8' requires target feature 'avx2', but would be inlined into function 'ProcessAlpha_SSE' that is compiled without support for 'avx2'
    __m128i solidCmp = _mm_broadcastb_epi8( a );
                       ^
3 warnings and 1 error generated.
error: command '/usr/bin/gcc' failed with exit code 1
Error: Process completed with exit code 1.
```
[complete log](https://github.com/K0lb3/etcpak/runs/2108373160?check_suite_focus=true)

Since this error doesn't seem to be related to the part I slightly modified I thought that you might be interested in this problem, as it would likely affect normal macOS users as well.

**Comment by wolfpld** [comment]
Fixed in 403d38b.

**Comment by K0lb3** [comment]
Thanks a lot.
